airbyte-integrations/connectors/source-stripe/integration_tests/abnormal_state.json

```diff
@@ -55,6 +55,13 @@
       "stream_descriptor": { "name": "customers" }
     }
   },
+  {
+    "type": "STREAM",
+    "stream": {
+      "stream_state": { "customer_updated": 10000000000 },
```

Comment:
Should the state be per partition here? It seems like we are at risk of losing records if it's not

Reply:
> Should the state be per partition here?

I don't think so because `ParentIncrementalStipeSubStream` class is used here. Thus, `Customers` are being synced incrementally and we'll get `payment_methods` only for the updated ones.

Reply:
Oh! So we have validated that when a payment method is updated, the client updated field is also updated. Can we document this somewhere?

Reply:
Also for the incremental part, shouldn't we rely on `payment_method.*` events instead? https://docs.stripe.com/api/events/types#event_types-payment_method.attached

Reply:
if we did validate the assumption, can it be documented in the code so we have a reference? else we'll assume we're at risk of losing records

Reply:
> So we have validated that when a payment method is updated, the client updated field is also updated.

Tested this, customer is NOT being updated on `payment_method` update.

>Also for the incremental part, shouldn't we rely on payment_method.* events instead?

Agreed, we should use this.


Moving PR to draft, need to change the implementation:
1st sync/Full refresh -- read from customers/{customer_id}/payment_methods
2nd sync/Incremental -- read from events stream with `type` = `payment_method.*`